composer.json

```diff
@@ -57,11 +57,43 @@
           "composer/installers": "^1.2.0"
         }
       }
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "ckeditor-plugin/link",
+        "version": "4.10.1",
+        "type": "drupal-library",
+        "dist": {
+          "url": "https://download.ckeditor.com/link/releases/link_4.10.1.zip",
+          "type": "zip"
+        },
+        "require": {
+          "composer/installers": "~1.0"
+        }
+      }
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "ckeditor-plugin/fakeobjects",
```

Comment:
What needs this library? What are you faking in the editor?

Reply:
its needed by the anchor link module https://www.drupal.org/project/anchor_link
not entirely sure what is does